src/evaluation/hit_at_k.py

```
@@ -73,6 +164,8 @@ def _hit_at_k(predictions, real_values):
         if count_mechane == 0:
             return 0
     except:
+        print(real_values)
```

please delete prints

---

better to raise an error